### DISCUSSION TITLE
CTkInputDialog

### DISCUSSION THREAD

**Discussion by NikishDAniel** [TomSchimansky/CustomTkinter#2183] [Q&A]
Can I able to take options from CTkInputDialog box?

**Comment by Anonymous6598** [comment]
Please, explain your question.

**Reply by NikishDAniel** [reply]
I've a button and it's command to show CTkInputDialog box , in that dialog i can able to enter the inputs, but it seems it have two options ok and cancel . onclicking ok it needs to do some actions , on cancel it needs to simply destroy that dialog box . so i need to take that option . that's it . but now i've created my own message popup box . THANKS for reply
